chassis/core/src/main/java/com/griddynamics/jagger/monitoring/MonitoringTask.java

```diff
@@ -44,7 +43,6 @@ public String getParentTaskId() {
     }
 
     @Override
-    @Required
```

Comment:
That is actually a bug which poped-up after I added "<context:annotation-config/>" (this line in Spring context activates Spring annotations like this one.). This property is declared as required but is never set in Spring context in all declarations. So, I just removed it.